src/libraries/Native/Windows/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -95,8 +95,6 @@ add_library(System.IO.Compression.Native-static
     ${NATIVECOMPRESSION_SOURCES}
 )
 
-SET_TARGET_PROPERTIES(System.IO.Compression.Native-static PROPERTIES OUTPUT_NAME System.IO.Compression.Native)
```

Comment:
This just isn't needed anymore, or it's being set somewhere else, or it was just wrong?

Reply:
This project is building both static and dynamic libraries on Windows. Both of these builds were producing `System.IO.Compression.Native.lib`. Building dynamic .dll produces .dll and export .lib; building static library produces .lib. The real .lib and the export .lib were overwriting each other that leads to build race conditions. We have run into it in the runtimelab NativeAOT branch. Deleting this line makes the names distinct and fixes the build race condition.

Reply:
Got it.